packages/client-utils/src/signing-smart-wallet-kit.ts

```diff
@@ -121,6 +202,18 @@ export const makeSigningSmartWalletKit = async (
      */
     executeOffer,
     sendBridgeAction,
+    /**
+     * Return an object-oriented view of a previously-saved result, with
```

Comment:
I don't totally understand the uses for this. Please add a test.

I gather that it's an entry à la "invokeEntry" but that's not clear from this jsdoc

Reply:
> I gather that it's an entry à la "invokeEntry" but that's not clear from this jsdoc

Correct; see https://github.com/Agoric/agoric-sdk/pull/12030#discussion_r2391114216 .

> I don't totally understand the uses for this. Please add a test.

Does that mean the design is worth pursuing?

Reply:
I assert this needs a test before landing.

It may need other validation of the design. I'll start a slack thread.